tailwind.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
         pink: "#FD346E",
         lightgray: "#767676",
         neutral: "#5A5A5A",
+        fontBlack: "#333333",
```

Comment:
keep it as #121212 as defined in wireframes

Reply:
have made the required change